Introduction.md

```diff
@@ -0,0 +1,44 @@
+ # What is Firedancer?
+- **[Firedancer](https://jumpcrypto.com/firedancer/) is a node implementation written in C for the solana blockchain**. It will be the second full node implementation of Solana with the comping up Light Client Tinydancer. Firedancer will diversify the client stack and make the network more resilient as if there is an bug in the original rust node that halts the validators from producing blocks and coming to a consensus, validators running FD will keep the network secure and stable.
+- ---
+# Why Another Validator?
+Firedancer will be solving some of the problems which solana as a network and community is trying to Solve.
+1. **Compute Fee and gas fee estimatation** -> Firedancers new and better blockpackaging algorithm does a lot better predicting the amount of Compute units used in a specific trnsaction making it easier for validators to choose which ones to include in the next block.When taken it makes 3 times less mistakes in predicting the CUs used. ![image](https://user-images.githubusercontent.com/88841339/224479879-466202fb-d085-4c93-beb6-22d6c3885a44.png)
+
+2. **Robustness**:When the validators are unable to reach consesnus and verify a transaction leading to a fork and downtime. One of the biggest memes floating around about Solana is its downtime, though it was a design tradeoff for it being the most performant blockchain, downtimes lead to a lot of different issues like liquidity, validator restarts etc which are mostly casued by small bugs in the updates in the core client code. 
+When there are 2 client implementations and one of them faces a bug, the nodes running the bug free code will continue processing transactions to which the other nodes could sync to later. 
+		
+ # How?
+ Firedancer has 3 high level Components which are modular and can be replaced with the components of Old Solana validator making it like different  components being stitched together while also writing a better documentation of the original validator client
+- Components->
+	- 	Packet Ingress
+	- 	Runtime
+	- 	Consensus
+
+ ![image](https://user-images.githubusercontent.com/88841339/224098753-4a62136a-aad7-4343-89b0-f8c0fd45a990.png)
```

Comment:
We will need to rewrite this cause its not accurate to the direction we are heading in.

Reply:
oh gotcha, btw I head kevin explain fd in the community call and he said yu folks were making components and sewing  it up to the old validator so thought this was accurate.
Feel free to edit